src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -426,14 +426,17 @@ has to be updated, then for MEG it could be considered to be a new session.
 
 ## Empty-room MEG recordings
 
-Empty-room MEG recordings capture the environment and system noise. Their
-collection is RECOMMENDED, before/during/after each session. This data is stored
-inside a subject folder named `sub-emptyroom`. The `session label` SHOULD be
-that of the date of the empty-room recording (e.g. `ses-YYYYMMDD`). The
-`scans.tsv` file containing the date/time of the acquisition SHOULD also be
-included. Hence, users will be able to retrieve the empty-room recording that
-best matches a particular session with a participant, based on date/time of
-recording.
+Empty-room MEG recordings capture the environmental and recording system's noise.
+Their collection is RECOMMENDED, before/during/after each session.
+This data MUST be stored inside a subject folder named `sub-emptyroom`.
```

Comment:
I am not sure about MUST, it seems too strong. When discussing this during the MEG BEP preparation, we realized that there are two widely used stragegies
1. some labs record an empty room every day, which are then shared by all researchers that record study-specific subjects that day
2. some labs make it the responsibility of the researcher to record a short empty room recording prior to each subject (e.g. while preparing the subject and explaining the task) 

The 1st case lends itself to the `sub-emtpyroom` scheme and dates are needed to link subject measurementsa to noise measurements. In the 2nd case the data is already naturally linked and making the proposed scheme obligatory complicates the data organization.

In the 2nd case I would store the noise measurements along with the subjects data, i.e. as `sub-001_task-somethingreal_meg.ds` and `sub-001_task-noise_meg.ds`, both under `sub-001`.

Reply:
Thanks for the context Robert, I have pushed a major change to account for this.